plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -806,7 +806,10 @@ fu_cros_ec_usb_device_write_firmware (FuDevice *device,
 								     firmware,
 								     section,
 								     error)) {
-				return FALSE;
+				g_debug ("failed to transfer section, trying another write\n");
+				fu_device_add_flag (device, FWUPD_DEVICE_FLAG_ANOTHER_WRITE_REQUIRED);
+				g_clear_error (error);
```

Comment:
typically what you should do if an error is expected to not be critical is have a local scope `g_autoptr error_local` and then use that.  However I'd suspect you also only want to do this `another_write_required` if it's a certain error type - so maybe catch that and check?

Reply:
I pushed a few fixes to wip/hughsie/cros_ec-error but i really want to see that GError->message first

Reply:
Hi @superm1,

I check now for G_USB_DEVICE_ERROR_NOT_SUPPORTED and clear the error only on that case.

Other kinds of failures we will proceed to fail here.